cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -102,84 +102,106 @@ write_files:
     owner: root:root
     content: |
       ${ExtraJson}
+  - path: /tmp/bootstrap.sh
```

Comment:
Have you checked if the replacement of `${!varname}` and `${varname}` works the same in this section? In the shellscript section `${varname}` is replaced with CFN value while `${!varname}` is replaced with `${varname}`

Reply:
Yes, checked launch template, ${!varname} is replaced with ${varname}, ${varname} is replaced by the actual value,
for example:
`export cookbook_version=${CookbookVersion} ` ==> `export cookbook_version=aws-parallelcluster-cookbook-3.2.0`

`
 echo ${!cookbook_version} | tee /opt/parallelcluster/.bootstrapped` ==> `echo ${cookbook_version} | tee /opt/parallelcluster/.bootstrapped`